azure-pipelines.yml

```diff
@@ -0,0 +1,104 @@
+resources:
+  repositories:
+  - repository: OpenAstronomy
+    type: github
+    endpoint: nmearl
+    name: OpenAstronomy/azure-pipelines-templates
+    ref: master
+
+
+stages:
+
+  - stage: Coverage
+    dependsOn: []  # Have stage run concurrently with others
+    jobs:
+    - template: run-tox-env.yml@OpenAstronomy
+      parameters:
+        coverage: codecov
+        envs:
+          # Perform a coverage test
+          - linux: py37-test
```

Comment:
This won't actually do a coverage test, because you are missing ``--cov specutils`` in the tox file. In fact I don't think it's useful to have a dedicated coverage test for packages that aren't the core package - for the core package there is a significant time penalty in testing for coverage, but for a smaller package like this you really should just run all the jobs with coverage testing as it won't make much of a difference. So I would suggest removing this stage, and instead just adding ``coverage: codecov`` for all the other stages and adding ``cov specutils`` to the pytest command in the tox file.